packages/typespec-test/test/widget_dpg/spec/client.tsp

```diff
@@ -0,0 +1,8 @@
+import "@azure-tools/typespec-client-generator-core";
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+
+// Some models from routes with suppressed visibility are still desired for custom public surface.
+@@access(WidgetService.NonReferencedModel, Access.public);
+@@usage(WidgetService.NonReferencedModel, Usage.input | Usage.output);
```

Comment:
We still need this kind of configuration as tcgc getAllModels will not return the orphan models if we don't have this. I think it's fine as they will be required to other languages as well ?